charts/openforms/values.yaml

```diff
@@ -545,6 +545,15 @@ nginx:
     # requests:
     #   cpu: 250m
     #   memory: 256Mi
+  staticFileServing:
+    # -- Enable static file serving directly from Nginx (requires OpenForms >= 3.3.0)
+    enabled: false
+    # -- Minimum required OpenForms version for static file serving (semver format)
+    minVersion: "3.3.0"
```

Comment:
why should this be configurable? 🤔 

The application (Open Forms) determines this, the helm chart can't change that, no?